src/node.h

```diff
@@ -231,6 +231,19 @@ NODE_EXTERN void RunAtExit(Environment* env);
   }                                                                           \
   while (0)
 
+// Variation on NODE_DEFINE_CONSTANT that sets a String value.
+#define NODE_DEFINE_STRING_CONSTANT(target, constant)                         \
+  do {                                                                        \
+    v8::Isolate* isolate = target->GetIsolate();                              \
+    v8::Local<v8::String> constant_name =                                     \
+        v8::String::NewFromUtf8(isolate, #constant);                          \
+    v8::Local<v8::String> constant_value =                                    \
+        v8::String::NewFromUtf8(isolate, constant);                           \
+    v8::PropertyAttribute constant_attributes =                               \
+        static_cast<v8::PropertyAttribute>(v8::ReadOnly | v8::DontDelete);    \
+    (target)->ForceSet(constant_name, constant_value, constant_attributes);   \
```

Comment:
Hmm... as far as I can tell, Set() doesn't allow setting the read-only property attribute so I'm not sure how that works (every time I think I understand this part of V8, they go and change it!). I did see in the current V8 checked into master that the maybe<bool> version of ForceSet is preferred so I've switch to that for now, but can switch to Set() once I understand how to use it to set a constant read-only value.